tests/features/general.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Miscellaneous general settings unit tests
+
+  @ubi8/openjdk-8
+  @ubi8/openjdk-8-runtime
+  @ubi8/openjdk-11
+  @ubi8/openjdk-11-runtime
+  @ubi8/openjdk-17
+  @ubi8/openjdk-17-runtime
```

Comment:
these @-prefixed lines select which images the tests are run against. Here we specify only the ubi8-based ones. I think that this test should run against all the images. The way the tag matching works, if you specify `@ubi8`, then any image beginning wtih  `@ubi8` (i.e. all of these ones) would match. So these lines could be replaced with just `@ubi8`. But please also add lines `@redhat-openjdk-18/openjdk18-openshift` and `@openjdk/openjdk-11-rhel7` to match our other two images.